### ISSUE TITLE
Multiple calendar parsing

### ISSUE THREAD

**Issue by mech** [icalendar/icalendar#157]
Hey, I have 2 years worth of public holidays to import. In the last version of icalendar, I can make use of the custom properties `x_wr_calname` to separate the 2 years and know for sure which year is for which year, but now the latest version the custom properties actually merges everything it finds into a single array like this:

`{"x_wr_calname"=>["Public Holidays 2016", "Public Holidays 2017"]`

It seems the newer version merges every `BEGIN:VCALENDAR` rather than separate them, is that true?

My ICS is as follow:

```
BEGIN:VCALENDAR
--- VEVENTs here... ---
END:VCALENDAR

BEGIN:VCALENDAR
--- VEVENTs here... ---
END:VCALENDAR
```

As you can see, I have 2 VCALENDAR markup here to represent 2 separate years of public holidays. But somehow icalendar 2.4.0 will merge them together.

**Comment by schnika** [comment]
I think this is related to #152. This should be fixed in https://github.com/icalendar/icalendar/pull/156

**Comment by rahearn** [comment]
Fixed by #156
